lyluatex.md

```diff
@@ -319,6 +319,19 @@ With this default option each score is compiled as a sequence of PDF files
 representing one system each. By default the systems are separated by a
 paragraph and a variable skip depending on the staffsize.
 
+\lyIssue{Note:}
+\option{insert=systems} implies the use of `lilypond-book-preamble.ly`.
+It is worth pointing out that this will *not* have any notion of pages
```

Comment:
Minor detail: I'd replace "this" with "the score" anymore to make it more clear.

I suggest "pages and page breaking anymore"